build/Targets/Packages.props

```diff
@@ -82,10 +82,12 @@
     <MicrosoftVisualStudioGraphModelVersion>15.0.26730-alpha</MicrosoftVisualStudioGraphModelVersion>
     <MicrosoftVisualStudioImageCatalogVersion>15.0.26730-alpha</MicrosoftVisualStudioImageCatalogVersion>
     <MicrosoftVisualStudioImagingVersion>15.0.26730-alpha</MicrosoftVisualStudioImagingVersion>
-    <MicrosoftVisualStudioImagingInterop140DesignTimeVersion>14.3.25407</MicrosoftVisualStudioImagingInterop140DesignTimeVersion>
+	<!-- This next line seems weird -->
+    <MicrosoftVisualStudioImagingInterop140DesignTimeVersion>15.0.25726-Preview5</MicrosoftVisualStudioImagingInterop140DesignTimeVersion>
```

Comment:
I'm using `microsoft.internal.visualstudio.shell.interop.14.1.designtime\14.0.25023\lib\Microsoft.Internal.VisualStudio.Shell.Interop.14.1.DesignTime.dll` to get `IVsExperimentationService`, which provides the A\B test and feature flag functionality. Other than that clue, I'm not sure why nuget bumps this.